activemodel/test/cases/errors_test.rb

```diff
@@ -162,6 +162,24 @@ def test_no_key
     assert_equal ["cannot be blank"], person.errors[:name]
   end
 
+  test "add an error message with interpolated %{model}" do
+    person = Person.new
+    person.errors.add(:name, :blank, message: "%{model}")
+    assert_equal ["Person"], person.errors[:name]
+  end
+
+  test "add an error message with interpolated %{attribute}" do
+    person = Person.new
+    person.errors.add(:name, :blank, message: "%{attribute}")
+    assert_equal ["name"], person.errors[:name]
+  end
+
+  test "add an error message with interpolated %{value}" do
+    person = Person.new
+    person.errors.add(:name, :blank, message: "%{value}")
+    assert_equal ["nil"], person.errors[:name]
```

Comment:
`nil` shows us that '%{value}' is indeed interpolated and that it's `value.inspect` (not `value`), so it's completely valid usage as a test.
It can be replaced w/ test that looks more meaningful, but I don't really see an issue here.